cli/src/declarations.ts

```diff
@@ -588,12 +588,15 @@ export interface LiveUpdateConfig {
   appId: string;
   channel: string;
   autoUpdateMethod: AutoUpdateMethod;
+  strategy: LiveUpdateStrategy;
```

Comment:
Did we want to make this optional and default to 'differential' per the conversation we had with Netkow?

Reply:
YES, thanks! Fixed.